config/locales/en.yml

```diff
@@ -2868,6 +2868,41 @@ en:
+          foreign_accounts_owed: International ACH payments are not allowed.
+          foreign_accounts_refund: International ACH direct deposit refunds are not allowed.
+          routing_number: Routing Number
+          withdraw_amount: How much do you authorize to be withdrawn from your account? (Your total amount due is $%{owed_amount}.)
+      nc_taxes_owed:
+        edit:
+          automatic_draft_payments_info_one: Some savings, money market, and controlled spending accounts are not eligible for automatic drafts. Please confirm with your bank or credit union if the account you’ve selected can be automatically drafted.
+          automatic_draft_payments_info_two: If your payment is returned from your bank unpaid, you will be subject to a "Penalty for Bad Electronic Funds Transfer." The penalty is 10% of the payment (minimum $1.00; maximum $1,000.00).
+          benefits_of_ncdor: Benefits of paying online (either here on FileYourStateTaxes or through NCDOR's website) include the ability to schedule payments and the security of knowing your payment has been received.
+          direct_debit_payment_info_four: If your payment is returned from your bank unpaid, you will be subject to a Penalty for Bad Electronic Funds Transfer. The penalty is 10% of the payment (minimum $1.00; maximum $1,000.00).
+          direct_debit_payment_info_one: The payment draft date must be a valid banking date, which does not include weekends or bank holidays. When entering your draft date, you will be provided with the earliest possible date.
+          direct_debit_payment_info_three: Your payment will be considered timely if you submit your payment on the due date and select the first available draft date. You will receive an NCDOR confirmation email after your payment is received.
+          direct_debit_payment_info_two: The draft date is based on a 5:00 pm EST cut off. In other words, if you submit your bank draft payment after 5:00 pm EST, the earliest draft date available will be two business days in the future.
+          how_would_you_like_to_pay: How would you like to pay?
+          more_info_about_automatic_draft_payments: More information about automatic draft payments
+          more_info_about_direct_debit_payment: More information about the direct debit payment date
+          page_title: You owe state taxes. How would you like to pay?
+          pay_bank: Pay directly from your checking or savings account (direct debit)
+          pay_mail_online_html: Pay by mail or online at <a target="_blank" rel="noopener nofollow" href="https://electronic-services.dor.nc.gov/wps/portal/d400v">NCDOR.gov</a>
+          payment_options: You can pay directly from your checking or savings account here. You can also pay by mail or online through the North Carolina Department of Revenue’s website.
+          title_html: You owe <strong>$%{owed_amount}</strong> in %{state_name} state taxes.
```

Comment:
should we remove this `%{state_name}` interpolation now that this is only getting used in North Carolina? [dust]